CMakeLists.txt

```
@@ -142,30 +142,52 @@ include(CPack)
 find_package(LibXml2 REQUIRED)
 find_package(QuickHull REQUIRED)
 
-if(USE_HRPSYSUTIL)
-  find_package(SDL REQUIRED)
-  find_package(OpenGL REQUIRED)
-  find_package(GLUT REQUIRED)
-  pkg_check_modules(OpenCV opencv)
+set(USE_HRPSYSUTIL ON)
+find_package(SDL REQUIRED)
+if (NOT SDL_FOUND)
+  message(WARNING "SDL is required to build hrpsysUtil")
+  set(USE_HRPSYSUTIL OFF)
+endif()
+find_package(OpenGL REQUIRED)
+if (NOT OpenGL_FOUND)
```

```suggestion
if (NOT OPENGL_FOUND)
```
OpenGL_FOUNDではなくOPENGL_FOUNDに値がセットされるようです。

https://github.com/fkanehiro/hrpsys-base/pull/1310/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL166

---

i think default behavior of `find_packaeg(<package` is to set `<package>_FOUND`
https://cmake.org/cmake/help/v3.0/command/find_package.html

but
```
$ cat CMakeLists.txt 
cmake_minimum_required(VERSION 2.4)
project(hrpsys)
find_package(OpenGL REQUIRED)
message("=> OpenGL_FOUND: ${OpenGL_FOUND}")
message("=> OPENGL_FOUND: ${OPENGL_FOUND}")
```
outputs 
```
-- Found OpenGL: /usr/lib/x86_64-linux-gnu/libOpenGL.so   
=> OpenGL_FOUND: TRUE
=> OPENGL_FOUND: TRUE
```
???

---

`find_package(OpenGL)`をすると`FindOpenGL.cmake`が実行されるのですが、

[FindOpenGL.cmakeの中でFIND_PACKAGE_HANDLE_STANDARD_ARGS関数が呼ばれる](https://github.com/Kitware/CMake/blob/867be20c80c8afe41727a8f8862474ef87571a6b/Modules/FindOpenGL.cmake#L443)と、FIND_PACKAGE_HANDLE_STANDARD_ARGS関数の中で[OpenGL_FOUNDとOPENGL_FOUND両方がセットされる](https://github.com/Kitware/CMake/blob/867be20c80c8afe41727a8f8862474ef87571a6b/Modules/FindPackageHandleStandardArgs.cmake#L600-L601)ようです。

ところが、cmake 2.4.7を使っているような古いPCでは、FindOpenGL.cmakeの中でFIND_PACKAGE_HANDLE_STANDARD_ARGSが呼ばれず、[OPENGL_FOUNDだけを直接セットしている](https://github.com/Kitware/CMake/blob/0846bdba097621b8982b73923c18e1b2fb6370eb/Modules/FindOpenGL.cmake#L140)ようです。

そのため、古いcmakeと今のcmake両方で動くようにするためには、`OpenGL_FOUND`ではなく`OPENGL_FOUND`を使う必要があるのではないかと思います。

---

thank you. please create PR.

---

done